package.json

```diff
@@ -23,7 +23,7 @@
         "prebuild": "npm run clean",
         "prepublishOnly": "node scripts gen",
         "postpublish": "node scripts clean",
-        "build": "npm run build:css && npm run lint:js && npm run copy",
+        "build": "npm run copy:svgToDocs && npm run build:css && npm run lint:js && npm run copy",
```

Comment:
Can we just add this copy:svgToDocs to "copy" npm command? Might make more sense to put it there. 
Especially since copy is already called from "build" command.